unit_tests/longevity/conftest.py

```diff
@@ -0,0 +1,30 @@
+import unittest.mock
+
+import pytest
+
+from sdcm import sct_config
+from sdcm.sct_events import events_processes
+
+
+@pytest.fixture(scope="function", autouse=True)
+def fixture_mock_calls(monkeypatch):
+    with unittest.mock.patch("sdcm.tester.validate_raft_on_nodes"):
+        yield
+
+    # clear the events processes registry after each test, so next test would be ableto start it fresh
+    events_processes._EVENTS_PROCESSES = None
+
+
+@pytest.fixture(scope="function", autouse=True)
+def params(request: pytest.FixtureRequest, monkeypatch):
+    monkeypatch.delenv("HOME", raising=False)
```

Comment:
Those can be removed, it was introduced cause of using pytester

Reply:
Removed the fixture and instead rely on the same fixture in root conftest.py